src/components/search/GlobalSearchField.js

```diff
@@ -375,12 +463,17 @@ function GlobalSearchField(props) {
                         id: searchConstants.DATA,
                     };
                 });
-                const viewAll = {
-                    id: searchConstants.VIEW_ALL_ID,
-                    name: searchTerm,
-                    resultType: { type: t("data"), id: searchConstants.DATA },
-                };
-                setOptions([...options, ...data, viewAll]);
+                if (filter === searchConstants.DATA) {
+                    setOptions([
+                        ...options,
+                        ...data,
+                        viewAllDataOptions,
+                        viewAllAppOptions,
+                        viewAllDataOptions,
```

Comment:
This last one is a dupe - guessing it should probably be `viewAllAnalysesOptions`?